packages/iov-bcp/src/transactions.ts

```diff
@@ -164,6 +164,20 @@ export function isAmount(data: any): data is Amount {
   );
 }
 
+/** A general interface for blockchain fees */
+export interface Fee {
+  readonly tokens?: Amount;
+  readonly gasPrice?: Amount;
+  readonly gasLimit?: Amount;
+}
+
+export function isFee(data: any): data is Fee {
+  return (
+    (typeof data === "object" && data !== null && isAmount(data.tokens)) ||
+    (isAmount(data.gasPrice) && isAmount(data.gasLimit))
```

Comment:
the way this is grouped now should lead to a crash for `data` that is no object, since `data.gasPrice` is referenced after the first line returns false.

Reply:
Oh! Not sure how this happened, will fix.

Reply:
Most likely the currect CI fail is because of this